src/dumpsxiso/cdreader.cpp

```diff
@@ -321,6 +321,36 @@ void cd::IsoDirEntries::ReadDirEntries(cd::IsoReader* reader, int lba, int secto
 		});
 }
 
+void cd::IsoDirEntries::ReadDirEntriesSkip(cd::IsoReader* reader, int lba, int sectors)
+{
+	size_t numEntries = 0; // Used to skip the first two entries, . and ..
```

Comment:
Update - I just noticed it's the same in `cd::IsoDirEntries::ReadDirEntries` and you just copied it, my apologies - in this case you can leave it as-is and I'll check it at my own discretion, since I feel like I may have made a mistake there (depending on what the spec says).

Reply:
Update2 - looks like the only difference between `ReadDirEntries` and `ReadDirEntriesSkip` is then `!(entry.value().entry.flags & 0x2))`? You could just introduce a new boolean parameter to `ReadDirEntries` then instead of duplicating the entire function.

Reply:
Yeah, in retrospect, I should've just made it a boolean parameter, I'll do that